libs/src/sdk/oauth/OAuth.ts

```diff
@@ -171,7 +215,15 @@ export class OAuth {
     }, 500)
   }
 
-  renderButton(element: HTMLElement, options?: ButtonOptions) {
+  renderButton({
+    element,
+    scope = 'read',
```

Comment:
wondering if scope should be a string or an array of several scopes for future proofing? I suppose it could be a comma separated list in the future too

Reply:
changed to allow array (better for future) or string with single scope option (better for present)